core/src/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -266,32 +268,34 @@ public static void insulators(int x, int y, int x2, int y2, Cons<Tilec> iterator
         public void placed(){
             if(net.client()) return;
 
-            Boolf<Tilec> valid = other -> other != null && other != tile && ((!other.block().outputsPower && other.block().consumesPower) ||
-                (other.block().outputsPower && !other.block().consumesPower) || other.block() instanceof PowerNode) && linkValid(this, other)
-                && !other.proximity().contains(this) && other.power().graph != power.graph;
-
-            tempTileEnts.clear();
-            Geometry.circle(tile.x, tile.y, (int)(laserRange + 2), (x, y) -> {
-                Tilec other = world.ent(x, y);
-                if(valid.get(other)){
-                    if(!insulated(this, other)){
-                        tempTileEnts.add(other);
+            if(Core.settings.getBool("autonodeconnection")) {
```

Comment:
This won't work on servers. Automatic node connection is done server-side to prevent spam of packets from the client.

Reply:
So i should edit server-side?

Reply:
You *did* edit server-side; what I'm saying is, when a client disables node connections on their end, it won't change how the server behaves.

Reply:
Then how to implement it?